packages/auth/src/__tests__/AuthProvider.test.tsx

```diff
@@ -52,6 +54,17 @@ const AuthConsumer = () => {
     error,
   } = useAuth()
 
+  const [authToken, setAuthToken] = useState(null)
+
+  const retrieveToken = async () => {
+    const token = await getToken()
+    setAuthToken(token)
+  }
+
+  useEffect(() => {
+    retrieveToken()
+  })
```

Comment:
So this will run on every re-render, is that the intention?

Reply:
I hadn't looked through the test in detail to be honest, but it doesn't hurt for it to run on rerender. It's there mainly because getting the token is now async.

Reply:
Maybe we should just add a blank array so that it's only run on mount?